.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: go
+before_script:
+  - go get github.com/google/go-querystring/query
+  - go get github.com/pborman/uuid
```

Comment:
I'm wondering if we should use dep here?

Reply:
That's indeed a possibility.

Reply:
Can you try to see if it's not an hassle with travis-ci? If it's a pain, I'll just merge it as is ❤️

Reply:
I've successfully vendored the dependencies, tests are passing.